src/libstd/lib.rs

```diff
@@ -471,6 +471,13 @@ pub mod time;
 mod sys_common;
 mod sys;
 
+#[unstable(feature = "demangle_for_rustc", issue = "0")]
+#[doc(hidden)]
+pub mod demangle_for_rustc {
+    pub use sys_common::backtrace::demangle;
+    pub use sys_common::backtrace::PrintFormat;
+}
```

Comment:
IMO it is a shame that we’re exposing more unstable stuff from libstd. Would it be possible to extract the functionality into a crate and make both libstd and librustc_trans use that?